can/io/logger.py

```diff
@@ -40,6 +40,14 @@
 from .printer import Printer
 from .sqlite import SqliteWriter
 from .trc import TRCWriter
+from .pcapng import PcapngWriter
+
+
+try:
+    import pyzstd
+except ImportError:
+    pyzstd = None
```

Comment:
Zstandard addition should be a distinct commit. Please let me know if you would like to merge with squashing so that I can split it out into a new PR